app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryTask.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import TasksList from '../task/TasksList';
+import PropTypes from 'prop-types';
+
+class ExpandedStoryTask extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      task: ''
+    };
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onHandleSubmit = this.onHandleSubmit.bind(this);
+  }
+
+  onInputChange(e) {
+    this.setState({
+      task: e.target.value
+    });
+  }
+
+  onHandleSubmit() {
+    const { onSave } = this.props;
+
+    const task = this.state.task;
+    onSave(task);
+    this.setState({
+      task: ''
+    });
+  }
+
+  render() {
+    const { story, onUpdate, onDelete } = this.props;
+
+    return (
+      <div>
+        <div className="Story__section">
+          <div className="Story__section-title">
+            <div className="Story__section-title">
+              {I18n.translate('story.tasks')}
+            </div>
+          </div>
+        </div>
+        <div className="Story__list-task checkbox" >
+          <TasksList
+            tasks={story.tasks}
+            onDelete={(taskId) => onDelete(taskId)}
```

Comment:
Here you can use just 

```suggestion
            onDelete={onDelete}
```